Formula/libgpg-error.rb

```diff
@@ -18,7 +19,24 @@ class LibgpgError < Formula
     sha256 x86_64_linux:  "b862b10316313dabc9ddc7774e37c41c0700b886d361b86a1485052287b73fd0"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext" => :build
```

Comment:
Do we really need `gettext`? This is getting picked up as a runtime dependency.

Reply:
`autogen.sh` needs gettext. another option to avoid this dependency would be to patch `configure`. if you want i can do that instead, patch won't be as clean I think.

Reply:
I have made the patch directly to `configure` instead. This way we don't need to add any additional build dependencies.

Reply:
also removed changes to libgpg-error (i included them in this PR by mistake).